pkg/kubelet/BUILD

```diff
@@ -12,12 +12,12 @@ go_library(
         "active_deadline.go",
```

Comment:
It'd be great if we can have separate commits for moving/renaming files, so that the changes are easy to track. I know it's hard sometimes though